src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const ul = document.querySelector('.gallery__list');
+const largeImg = document.getElementById('largeImg');
+
+ul.onclick = function(events) {
```

Comment:
```suggestion
ul.onclick = function(event) {
```
тебе приходит информация об одном объекте